mllib/src/main/scala/org/apache/spark/ml/fpm/FPGrowth.scala

```diff
@@ -217,9 +217,13 @@ object FPGrowth extends DefaultParamsReadable[FPGrowth] {
 @Experimental
 class FPGrowthModel private[ml] (
     @Since("2.2.0") override val uid: String,
-    @Since("2.2.0") @transient val freqItemsets: DataFrame)
+    @Since("2.2.0") @transient val freqItemsets: DataFrame,
+    private val itemSupport: Map[Any, Long])
   extends Model[FPGrowthModel] with FPGrowthParams with MLWritable {
 
+  private[ml] def this(uid: String, freqItemsets: DataFrame) =
```

Comment:
Because it's private, you could also just make the new parameter an Option or something rather than recreate the existing constructor separately.